### PR TITLE
Update PR preview (purge older files, add documentation)

### PR DESCRIPTION
**Pull request by gwarf**
Update PR preview (purge older files, set more options explicitly, add documentation)

### CODE REVIEW

File: .github/workflows/deploy_pr_preview.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+      - name: Extract pull request preview
+        run: unzip pr.zip
+
+      - name: Deploy Pull Request preview
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          # Use GITHUB_TOKEN allowing to write to local repository
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # XXX add a way to clean deployed PR having been closed or merged
```
Comment (enolfc) [inline .github/workflows/deploy_pr_preview.yml:49]: I guess this comment can be removed now

**Review by enolfc — APPROVED** [review]
ok!

File: .github/workflows/deploy_pr_preview.yml (hunk 1 above)
Comment (gwarf) [reply .github/workflows/deploy_pr_preview.yml]: yep, you are right. Let's see later if it works.

File: .github/workflows/deploy_pr_preview.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          commit_message: ${{ github.event.head_commit.message }}
+
+      - name: Update Pull Request
+        uses: actions/github-script@v4.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            var fs = require('fs');
+            var issue_number = Number(fs.readFileSync('./PR'));
+            const output = `#### Documentation preview deployed!\`
```
Comment (enolfc) [inline .github/workflows/deploy_pr_preview.yml:70]:
```suggestion
            const output = `#### Documentation preview deployed!
```

Comment (enolfc) [reply .github/workflows/deploy_pr_preview.yml:70]: Not 100% on how js treats strings, but that seems to not belong

**Pull request merged**